### PR TITLE
S02 Lessons and Impact evaluation

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: code_schemes/s02_dadaab_impact_made.json [hunk 1]
```diff
@@ -3,6 +3,38 @@
   "Name": "S02 DADAAB Impact Made",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "CodeID": "code-22aad8d4",
+      "CodeType": "Normal",
+      "DisplayText": "There are changes",
```
Comment (IsaackMwenda) [inline code_schemes/s02_dadaab_impact_made.json]:
```suggestion
      "DisplayText": "there are changes",
```

Comment (IsaackMwenda) [reply code_schemes/s02_dadaab_impact_made.json]: use small case everywhere

File: code_schemes/s02_dadaab_impact_made.json [hunk 2]
```diff
@@ -3,6 +3,38 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "Gender equality",
+      "NumericValue": 3,
+      "StringValue": "Gender_equality",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-dd264c5c",
+      "CodeType": "Normal",
+      "DisplayText": "other",
```
Comment (IsaackMwenda) [inline code_schemes/s02_dadaab_impact_made.json]:
```suggestion
      "DisplayText": "other theme",
```

Comment (IsaackMwenda) [reply code_schemes/s02_dadaab_impact_made.json]: we have another `        "DisplayText": "meta: other",
` so the pipeline will pick this as a duplicate. Rename to` other theme`

File: code_schemes/s02_dadaab_lessons_learnt.json [hunk 3]
```diff
@@ -3,6 +3,38 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "learnt a lot",
+      "NumericValue": 3,
+      "StringValue": "learnt_a_lot",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-e61e36f1",
+      "CodeType": "Normal",
+      "DisplayText": "other",
```
Comment (IsaackMwenda) [inline code_schemes/s02_dadaab_lessons_learnt.json]:
```suggestion
      "DisplayText": "other theme",
```

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]
Thanks, @DanielMwendwa, this mainly looks good with a few changes that I would like to re-review before we merge.

File: code_schemes/s02_dadaab_impact_made.json (hunk 2 above)
Comment (DanielMwendwa) [reply code_schemes/s02_dadaab_impact_made.json]: updated thanks

File: code_schemes/s02_dadaab_lessons_learnt.json (hunk 3 above)
Comment (DanielMwendwa) [reply code_schemes/s02_dadaab_lessons_learnt.json]: updated thanks

File: code_schemes/s02_dadaab_impact_made.json (hunk 1 above)
Comment (DanielMwendwa) [reply code_schemes/s02_dadaab_impact_made.json]: updated thanks

**Review by IsaackMwenda — APPROVED** [review]
Thanks

**Pull request merged**